test/python/transpiler/test_consolidate_blocks.py

```diff
@@ -407,6 +407,29 @@ def test_single_gate_block_outside_target_with_matching_basis_gates(self):
         expected.swap(0, 1)
         self.assertEqual(expected, pass_manager.run(qc))
 
+    def test_identity_unitary_is_removed(self):
+        """Test that a 2q identity unitary is removed without a basis."""
+        qc = QuantumCircuit(5)
+        qc.h(0)
+        qc.cx(0, 1)
+        qc.cx(0, 1)
+        qc.h(0)
+
+        pm = PassManager([Collect2qBlocks(), ConsolidateBlocks()])
+        print(pm.run(qc))
+        self.assertEqual(QuantumCircuit(5), pm.run(qc))
+
+    def test_identity_1q_unitary_is_removed(self):
+        """Test that a 1q identity unitary is removed without a basis."""
+        qc = QuantumCircuit(5)
+        qc.h(0)
+        qc.h(0)
+        qc.h(0)
+        qc.h(0)
+        pm = PassManager([Collect2qBlocks(), Collect1qRuns(), ConsolidateBlocks()])
+        print(pm.run(qc))
```

Comment:
debug print?

Reply:
Removed in: https://github.com/Qiskit/qiskit-terra/pull/9222/commits/3cf8961f813a7b299b25531972555d6bbd652813